pkg/query-service/utils/time.go

```diff
@@ -12,3 +13,17 @@ func Elapsed(funcName string, args ...interface{}) func() {
 		zap.S().Infof("func %s took %v with args %v", funcName, time.Since(start), args)
 	}
 }
+
+// GetZerosForEpochNano returns the number of zeros to be appended to the epoch time for converting it to nanoseconds
+func GetZerosForEpochNano(epoch int64) int64 {
```

Comment:
it is used somewhere ?

Reply:
No, I don't recall why I added it here. Since you already added other similar function to utils. I think it can be removed.

Reply:
Yeah we can remove it.